src/main/java/com/wasilewskyy/medical_clinic/exception/MedicalClinicExceptionHandler.java

```diff
@@ -0,0 +1,25 @@
+package com.wasilewskyy.medical_clinic.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+
+@ControllerAdvice
+public class MedicalClinicExceptionHandler {
+
+    @ExceptionHandler(PatientNotFoundException.class)
+    ResponseEntity<String> handlePatientNotFoundException(PatientNotFoundException ex) {
```

Comment:
Dobrze byloby zdefiniowac klase ErrorMessage np. niech ta klasa ma pola message oraz HTTP Status i zwracajmy nie
ResponseEntity<String>  <- bo to nam nie gwarantuje formatu JSON tylko zwracajmy ResponseEntity<ErrorMessage>